lib/restaurant.rb

```diff
@@ -0,0 +1,27 @@
+require 'order'
+
+class Restaurant
+  attr_reader :current_order  
+
+  def initialize(order = Order, menu:)
+    @menu = menu
+    @current_order = []
+    @order = order
+  end
+
+  def print_menu
+    menu.print_menu
+  end
+
+  def add_to_order(dish, quantity = 1)
+    raise "Cannot add to order: Item is not on menu" if not_on_menu?(dish) 
+    current_order << @order.new(dish, quantity)
+  end
+
+  private
+  attr_reader :menu, :order
```

Comment:
Are these needed in a private section? Anything in private still has accesses to the instance variables.